src/models/geograph.py

```diff
@@ -488,39 +492,70 @@ def merge_nodes(
             zip_longest([final_index], adjacency_set, fillvalue=final_index)
         )
 
+    def merge_classes(
+        self, class_list: List[Union[str, int]], new_name: Union[str, int]
+    ) -> None:
+        """
+        Merge multiple classes together into one by renaming the class labels.
+
+        Args:
+            new_name (Union[str, int]): The new name for the combined class,
+            either a string or an int.
+            class_list (List): The list of names of class labels to combine.
+            Every name in the list must be in the GeoGraph.
+
+        Raises:
+            ValueError: If `class_list` contains a class name not already in
+            the GeoGraph.
+        """
+        if not set(class_list).issubset(self.df["class_label"].unique()):
+            raise ValueError("`class_list` must only contain valid class names.")
+        self.df.loc[self.df["class_label"].isin(class_list), "class_label"] = new_name
+
     def add_habitat(
         self,
         name: str,
-        valid_classes: List[int],
+        valid_classes: List[Union[str, int]],
+        barrier_classes: Optional[List[Union[str, int]]] = None,
         max_travel_distance: float = 0.0,
```

Comment:
Nice to have this now 🚀

Reply:
Does this actually go into the computation yet, or is this just for future-proofing?

Reply:
It's just for future proofing at the moment.